.github/issue-template.md

```diff
@@ -1,25 +1,23 @@
 <!--
-Uncomment leaving one or more of the following to ensure that the appropriate
-working groups are aware of the issue:
+Pro-tip: You can leave this block commented, and it still works!
+
+Select the appropriate areas for your issue and remove the leading >
```

Comment:
You got rid of the leading `>`!!!

Reply:
I was testing you! You passed. 👍 

(Thanks for catching that)